src/components/Select/MultiSelect.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import ChevronDownIcon from '@moda/icons/chevron-down-12';
+import ChevronUpIcon from '@moda/icons/chevron-up-12';
+import ExitIcon from '@moda/icons/exit-16';
+import { SelectableOption } from '.';
+import { SelectOptions } from './SelectOptions';
+import { useSelect } from './useSelect';
+import './MultiSelect.scss';
+
+type Props = Omit<React.HTMLAttributes<HTMLDivElement>, 'defaultValue' | 'value' | 'onChange'> & {
+  defaultValue?: string[] | undefined;
+  disabled?: boolean;
+  idRef?: string;
+  name?: string;
+  placeholder?: string;
+  onChange?: (value: string[]) => void;
+  options: SelectableOption[];
+  value?: string[] | undefined;
+  error?: boolean | string;
+  shiftIconLeftwards?: boolean;
+  ignoreCasing?: boolean;
+  dropDirection?: 'down' | 'up';
+};
+
+export const MultiSelect: React.FC<Props> = ({
+  idRef = '',
+  className,
+  options,
+  name,
+  disabled,
+  value,
+  placeholder,
+  onChange,
+  defaultValue,
+  error,
+  shiftIconLeftwards = false,
+  ignoreCasing = false,
+  dropDirection = 'down',
+  ...rest
+}) => {
+  const { state, dispatch, Mode, selectRef } = useSelect({ value, defaultValue });
+  const [searchPhrase, setSearchPhrase] = useState('');
+
+  const stateValueArray = useMemo(
+    () => (typeof state.value == 'object' ? state.value : []),
+    [state]
+  );
+
+  const handleSelect = useCallback(
+    (option: SelectableOption) => {
+      dispatch({ type: 'SELECT', payload: { value: option.value } });
+      setSearchPhrase('');
+      onChange && onChange(stateValueArray ? [...stateValueArray, option.value] : [option.value]);
```

Comment:
`stateValueArray` should always be an array right? so this check seems unnecessary

Reply:
Correct, good catch thanks